bin/run_bam_qc.py

```diff
@@ -64,11 +67,18 @@ def validate_args(args):
         valid = valid and validate_output_dir(parent_path)
     if args.temp_dir != None:
         valid = valid and validate_output_dir(args.temp_dir)
+    if args.workflow_version != None:
+        # version string should be of the form 0.12.34 or (for instance) 0.12.34_alpha
+        pattern = '^[0-9]+\.[0-9]+\.[0-9]+\w*$'
```

Comment:
My IDE gives me a warning on this line, although it seems to work OK. I think it refers to this issue: https://stackoverflow.com/questions/50504500/deprecationwarning-invalid-escape-sequence-what-to-use-instead-of-d

I think adding `r'^[0-9]etc` makes the warning go away.

Reply:
OK, will fix.